netbox/dcim/fields.py

```diff
@@ -11,6 +11,11 @@ class ASNField(models.BigIntegerField):
         MaxValueValidator(4294967295),
     ]
 
+    def formfield(self, *args, **kwargs):
```

Comment:
Minor nit, but `formfield()` doesn't take positional args; only kwargs.

Reply:
My bad. I added it for "completeness" but I should've just stuck to what the documentation said.